scripts/procdockerstatsd

```diff
@@ -168,6 +168,30 @@ class ProcDockerStats(daemon_base.DaemonBase):
                 cmd = row.get('CMD')
                 self.update_state_db(value, 'CMD', cmd)
 
+    def update_procfipsstats_command(self):
+        fips_enable_flag = '0'
+        current_enforce_state = '0'
+        next_enforce_state = '0'
+        if os.path.isfile('/etc/fips/fips_enable.flag'):
+            with open('/etc/fips/fips_enable.flag') as f:
+                fips_enable_flag = f.read().trip()
+        with open('/proc/cmdline') as f:
+            cmdline = f.read().trip()
+            if 'fips=1' in cmdline:
+                current_enforce_state = '1'
+        cmd = ['sonic-installer', 'get-fips']
+        exitcode, data = getstatusoutput_noshell(cmd)
+        if any(exitcode):
+            self.log_error("Error running command '{}'".format(cmd))
+            data = None
+        if 'enabled' in data:
+            next_enforce_state = '1'
+        fips_db_key = 'FIPS_STATS|state'
+        self.state_db.delete_all_by_pattern('STATE_DB', "FIPS_STATS|*")
```

Comment:
>delete_all_by_pattern [](http://example.com/codeflow?start=22&length=21)

Do not delete, it will impact subscribing.

Reply:
Fixed, thanks.